device.go

```diff
@@ -284,42 +292,37 @@ func updateSpecDeviceList(device pb.Device, spec *pb.Spec) error {
 	}).Info("handling block device")
 
 	// Update the spec
-	for idx, d := range spec.Linux.Devices {
-		if d.Path == device.ContainerPath {
-			hostMajor := spec.Linux.Devices[idx].Major
-			hostMinor := spec.Linux.Devices[idx].Minor
-			agentLog.WithFields(logrus.Fields{
-				"device-path":        device.VmPath,
-				"host-device-major":  hostMajor,
-				"host-device-minor":  hostMinor,
-				"guest-device-major": major,
-				"guest-device-minor": minor,
-			}).Info("updating block device major/minor into the spec")
-
-			spec.Linux.Devices[idx].Major = major
-			spec.Linux.Devices[idx].Minor = minor
-
-			// there is no resource to update
-			if spec.Linux == nil || spec.Linux.Resources == nil {
-				return nil
-			}
+	idxData, ok := devIdx[device.ContainerPath]
+	if !ok {
+		return grpcStatus.Errorf(codes.Internal,
+			"Should have found a matching device %s in the spec",
```

Comment:
Huh?  %s is filled in with ContainerPath here.  This message hasn't actually changed, just moved a bit.

Reply:
I was referring to the grammar. It looks grammatically wrong to me. I would find it clearer if it said something like `Should have found a guest device matching host device /dev/foo` rather than `Should have found a matching device /dev/foo`.